smart-contracts/Makefile.smartContracts

```diff
@@ -1 +1 @@
-smartContracts=contracts/Blocklist.sol contracts/Valset.sol contracts/Migrations.sol contracts/ValsetStorage.sol contracts/CosmosBridgeStorage.sol contracts/BridgeBank/CosmosWhiteList.sol contracts/BridgeBank/CosmosWhiteListStorage.sol contracts/BridgeBank/BankStorage.sol contracts/BridgeBank/CosmosBank.sol contracts/BridgeBank/Rowan.sol contracts/BridgeBank/BridgeToken.sol contracts/BridgeBank/EthereumWhitelist.sol contracts/BridgeBank/BridgeBank.sol contracts/BridgeBank/Pausable.sol contracts/BridgeBank/EthereumBankStorage.sol contracts/BridgeBank/PauserRole.sol contracts/BridgeBank/CosmosBankStorage.sol contracts/OracleStorage.sol contracts/interfaces/IBlocklist.sol contracts/interfaces/IBridgeToken.sol contracts/Oracle.sol contracts/CosmosBridge.sol contracts/Mocks/FailHardToken.sol contracts/Mocks/Erowan.sol contracts/Mocks/ReentrancyToken.sol contracts/Mocks/FakeERC20.sol contracts/Mocks/ManyDecimalsToken.sol contracts/Mocks/TrollToken.sol contracts/BridgeRegistry.sol
+smartContracts=contracts/Blocklist.sol contracts/Valset.sol contracts/ValsetStorage.sol contracts/CosmosBridgeStorage.sol contracts/BridgeBank/CosmosWhiteList.sol contracts/BridgeBank/CosmosWhiteListStorage.sol contracts/BridgeBank/BankStorage.sol contracts/BridgeBank/CosmosBank.sol contracts/BridgeBank/Rowan.sol contracts/BridgeBank/BridgeToken.sol contracts/BridgeBank/EthereumWhitelist.sol contracts/BridgeBank/BridgeBank.sol contracts/BridgeBank/Pausable.sol contracts/BridgeBank/EthereumBankStorage.sol contracts/BridgeBank/PauserRole.sol contracts/BridgeBank/CosmosBankStorage.sol contracts/OracleStorage.sol contracts/interfaces/IBlocklist.sol contracts/interfaces/IBridgeToken.sol contracts/Oracle.sol contracts/CosmosBridge.sol contracts/Mocks/FailHardToken.sol contracts/Mocks/Erowan.sol contracts/Mocks/ReentrancyToken.sol contracts/Mocks/FakeERC20.sol contracts/Mocks/ManyDecimalsToken.sol contracts/Mocks/TrollToken.sol contracts/BridgeRegistry.sol
```

Comment:
Should these be committed?

Reply:
Yup, we no longer need migrations since moving from truffle.